xmake/rules/c++/modules/modules_support/clang.lua

```diff
@@ -68,37 +68,43 @@ end
 function load(target)
     -- get module and module cache flags
     local modulesflag = get_modulesflag(target)
-    local builtinmodulemapflag = get_builtinmodulemapflag(target)
     local implicitmodulesflag = get_implicitmodulesflag(target)
-    local noimplicitmodulemapsflag = get_noimplicitmodulemapsflag(target)
 
     -- add module flags
     target:add("cxxflags", modulesflag)
 
     -- add the module cache directory
     target:add("cxxflags", builtinmodulemapflag, {force = true})
     target:add("cxxflags", implicitmodulesflag, {force = true})
-    target:add("cxxflags", noimplicitmodulemapsflag, {force = true})
+    target:add("cxxflags", "-fvisibility=default", {force = true})
 
     target:data_set("cxx.modules.use_libc++", table.contains(target:get("cxxflags"), "-stdlib=libc++"))
+    target:add("ldflags", "-lc++")
```

Comment:
why? and it does not support shared library (shflags). 

clang may also use stdc++ and we should not hard code it.

Reply:
The same as the comment above